Add all meatmonsters.

gggritso opened the pull request:
I don’t have an API key to test everything with, but on my computer the regexes were all triggered correctly, so I think this is good to go.

gggritso committed:
Add all meatmonsters.

Also remove all skyhookers who aren’t Bryce, because … come on.

omni5cience commented:
First of all THANK YOU! I've been putting this off for way too long. However, there's a slight problem, several of the monsters (duane and trek in particular) have multiple of what I've been calling bots. It looks like your scraper merged those, so for example if I type `!help` instead of `zuul.gif` and "THERE IS NO HELP THERE IS ONLY ZUUL" I might get `excellent.gif` and "GO! GO! GO!"

gggritso committed:
Un-roll the mashed bots

This fixes an issue where multiple bots in the same folder weren’t
processed correctly.

gggritso committed:
Add moot bot!

I have no idea how it slipped through, sorry! I’m bad at the git.

gggritso commented:
:sparkles:
